tests/fixtures.ts

```diff
@@ -108,6 +108,49 @@ class MCPServer extends EventEmitter {
       this._child.stdin?.end();
     });
   }
+
+  async listTools() {
+    const list = await this.send({
```

Comment:
Your send can now be higher level and accept {method, params} while generating ids and inserting jsonrpc!